protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -584,6 +590,48 @@ definitions:
       - overwrite
       #- upsert_dedup # TODO chris: SCD Type 1 can be implemented later
       - append_dedup # SCD Type 1 & 2
+      - insert
+      - update
+      - upsert
```

Comment:
Is there a concrete difference between `upsert` and `append_dedup`?

Reply:
Notes from talk with @pedroslopez:

- Overwrite is certainly a distinct operation, because it clears data. Not a DA operation at all.
- insert seems a lot like append, we can't discern a meaningful difference
- upsert seems a lot like append_dedup, we can't discern a meaningful difference
- We should explore whether we can reuse `append` and `append_dedup` and just reskin them in the UI